languages/csharp/exercises/concept/expression-bodied-members/ExpressionBodiedMembersTests.cs

```diff
@@ -0,0 +1,137 @@
+using System;
+using Xunit;
+
+public class ExpressionBodiedMembersTests
+{
+    [Fact]
+    public void GetReading()
+    {
+        var ws = new WeatherStation();
+        ws.AcceptReading(new Reading(20m, 25m, 0.01m, WindDirection.Unknown));
+        decimal[] expected = {20, 25, 0.01m};
+        decimal[] actual = {ws.LatestTemperature, ws.LatestPressure, ws.LatestRainfall};
+        Assert.Equal(expected, actual);
+    }
+
+    [Fact]
```

Comment:
Could you add the `Skip` property to this test and the subsequent tests?

Reply:
The tests all pass from the start.  As you know with refactoring the tests keep you honest throughout the process.